lib/cli-engine.js

```diff
@@ -273,6 +273,17 @@ function CLIEngine(options) {
      */
     this.options = assign(Object.create(defaultOptions), options || {});
 
+    /**
+	 * fileCache used to not operate on files that haven't changed
+     * since last successful execution
+     * @type {Object}
+     */
+    this.fCache = require("file-entry-cache").create(path.resolve(".eslintcache"));
```

Comment:
yes it can be configured, but @nzakas suggested to use the current working directory, making it a parameter is rather trivial. @nzakas what do you think?